src/components/headerMobile/HomeHeaderMobile.jsx

```diff
@@ -0,0 +1,51 @@
+import "./homeHeaderMobile.css";
+import Restaurant from "../../static/images/restaurant.png";
+import ArrowDown from "../../static/images/arrow-down.svg";
+import Pin from "../../static/images/pin-o.svg";
+import HomeFilled from "../../static/images/home-filled.svg";
+import Explorer from "../../static/images/explorer.svg";
+import Order from "../../static/images/order.svg";
+import Profile from "../../static/images/profile.svg";
+
+function HomeHeaderMobile() {
+  return (
+    <header className="header-mobile">
+      <nav class="header-mobile__container">
+        <div class="header-mobile__icon-categories">
+          <img src={Restaurant} width="40" height="40" />
+        </div>
+        <div class="input-wrapper">
+          <span class="input-wrapper__title">Entregar em</span>
+          <div class="address-info">
+            <img src={Pin} width="15" height="15" />
+            <span class="input_wrapper__item">R.Sol, 1233</span>
+            <div class="image_icon">
+              <img src={ArrowDown} />
+            </div>
+          </div>
+        </div>
+      </nav>
+      <div class="tab-bar">
+        <div class="tab-bar__home">
+          <img src={HomeFilled} />
+          <span class="tab-bar__text">Inicio</span>
+        </div>
+        <div class="tab-bar__explorer">
+          <img src={Explorer} />
+          <span class="tab-bar__text">Buscar</span>
+        </div>
+        <div class="tab-bar__order">
+          <img src={Order} />
+          <span class="tab-bar__text">Pedido</span>
+        </div>
+        <div class="tab-bar__profile">
+          <span class="tab-bar__icon-profile">
+            <img src={Profile} />
```

Comment:
I think it would be good to add the alt tag in all images and make it clear what this image is the alt description

![image](https://user-images.githubusercontent.com/38358239/152177609-3e9951ba-0265-4c06-ac1a-1c1e0fcd786c.png)

Reply:
good point, I added to all the images <3

Reply:
just a tip, a visually impaired reader will hear the substitute text in place of the image, so always describe exactly what this image is. this is so important and I think we don't do it correctly in the ifood website.